.github/workflows/CI.yml

```
@@ -0,0 +1,67 @@
+name: CI
+on:
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.1'
+          - 'nightly'
+        os:
+          - ubuntu-latest
+          - macOS-latest
+          - windows-latest
```

I think test against Linux only is fine
```suggestion
          - ubuntu-latest
```

---

I do not know, windows and macos is free to test, kind of why not? It's green anyway.

---

well, just a trade-off in terms of the CI running time. Since  I don't gain extra information/different result from the win/macos builds in most of cases. (And I'm impatient :p